doc/deploy_starrocks_with_operator_howto.md

```diff
@@ -0,0 +1,297 @@
+starrocks-controller-65bb8679-jkbtg   1/1     Running   0          22h
+starrockscluster-sample-be-0          1/1     Running   0          23h
+starrockscluster-sample-be-1          1/1     Running   0          23h
+starrockscluster-sample-be-2          1/1     Running   0          22h
+starrockscluster-sample-fe-0          1/1     Running   0          21h
+starrockscluster-sample-fe-1          1/1     Running   0          21h
+starrockscluster-sample-fe-2          1/1     Running   0          22h
+```
+
+> **Note**
+>
+> If some pods cannot start after a long period of time, you can use `kubectl logs -n starrocks <pod_name>` to view the
+> log information or use `kubectl -n starrocks describe pod <pod_name>` to view the event information to locate the
+> problem.
+
+## 3. Manage StarRocks Cluster
+
+### 3.1. Access StarRocks Cluster
+
+The components of the StarRocks cluster can be accessed through their associated Services, such as the FE Service. For
+detailed descriptions of Services and their access addresses,
+see [api.md](https://github.com/StarRocks/starrocks-kubernetes-operator/blob/main/doc/api.md)
+and [Services](https://kubernetes.io/docs/concepts/services-networking/service/).
+
+```bash
+$ kubectl get svc
+NAME                                 TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)                               AGE
+starrockscluster-sample-fe-search    ClusterIP   None           <none>        9030/TCP                              76s
+starrockscluster-sample-fe-service   ClusterIP   10.96.26.146   <none>        8030/TCP,9020/TCP,9030/TCP,9010/TCP   76s
+```
+
+#### 3.1.1. Access StarRocks Cluster from within Kubernetes Cluster
+
+From within the Kubernetes cluster, the StarRocks cluster can be accessed through the FE Service's ClusterIP.
+
+1. Obtain the internal virtual IP address `CLUSTER-IP` and port `PORT(S)` of the FE Service.
+
+    ```Bash
+    $ kubectl -n starrocks get svc 
+    NAME                                 TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)                               AGE
+    starrockscluster-sample-be-search    ClusterIP   None           <none>        9050/TCP                              66s
+    starrockscluster-sample-be-service   ClusterIP   10.96.86.207   <none>        9060/TCP,8040/TCP,9050/TCP,8060/TCP   66s
+    starrockscluster-sample-fe-search    ClusterIP   None           <none>        9030/TCP                              2m27s
+    starrockscluster-sample-fe-service   ClusterIP   10.96.26.146   <none>        8030/TCP,9020/TCP,9030/TCP,9010/TCP   2m27s
+    ```
+
+2. Access the StarRocks cluster by using the MySQL client from within the Kubernetes cluster.
+
+   ```Bash
+   mysql -h 10.100.162.xxx -P 9030 -uroot
+   ```
+
+   Upon deploying a fresh StarRocks cluster, the `root` user's password remains unset, potentially posing a security
+   risk.
+   See [Init Root Password When First Deploy HOWTO](./initialize_root_password_howto.md) for details on how to set the
```

Comment:
here the cluster is deployed manually by operator with crd, better point to the manual change root password doc.